templates/zabbix_proxy.conf.erb

```diff
@@ -406,20 +406,20 @@ Include=<%= @include_dir %>
 #	Location of SSL client certificates.
 #	This parameter is used only in web monitoring.
 #
-<% if @sslcertlocation %>SSLCertLocation=<%= @sslcertlocation %><% end %>
```

Comment:
mhm where did those variables come from? is that something that needs to be cleaned up?

Reply:
They were added with support for zabbix proxy 3.0, but if you check the merge while the template was updated the params were never added, this is what I am fixing